src/main/java/com/sonyericsson/jenkins/plugins/bfa/ScanLogAction.java

```diff
@@ -0,0 +1,75 @@
+package com.sonyericsson.jenkins.plugins.bfa;
+
+import hudson.model.Action;
+import hudson.model.Run;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+
+/**
+ * The Action for adding a link to the analysis for each run.
+ */
+public class ScanLogAction implements Action {
```

Comment:
Should implement RunAction2 so you still have the association after a Restart of Jenkins